### PR TITLE
Update README to include `standard:fix` command

### PR DESCRIPTION
**Pull request by garyhtou**
### Describe your change in plain English.

To make the developer experience a little better, the README now includes instructions on how to run Standard's auto-fix command

### Link to the issue

https://github.com/rubyforgood/pet-rescue/issues/50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,22 @@ bin/dev
...
+rails test
+```
+
+### Running [Standard](https://github.com/standardrb/standard)
+
+We use Standard for linting. It provides a command for auto-fixing errors:
+
+```sh
+rails standard:fix
+```
```
Comment (edwinthinks) [inline README.md:56]: Could you put this below the "Accessing Roles" section? I think we want to show how to get the app running as fast as possible and then run tests.

**Review by edwinthinks — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (garyhtou) [reply README.md:56]: Yeah, great idea!

**Review by edwinthinks — APPROVED** [review]
